.github/release-please.yml

```diff
@@ -0,0 +1,3 @@
+manifest: true
+primaryBranch: master
+handleGHRelease: true
```

Comment:
If release please will create the tag, we probably need to drop the step in the CI build at https://github.com/microsoftgraph/msgraph-beta-sdk-dotnet/blob/3fb37dd971d96e0aa78f28909c9663eb848809a3/pipelines/ci-build.yml#L277

Reply:
Kindly recheck this. It's now updated.